src/components/GroupJoin/GroupsGallery.story.js

```diff
@@ -1,7 +1,7 @@
 import { storiesOf } from '@storybook/vue'
 import { action } from '@storybook/addon-actions'
 
-import GroupPreviewGallery from './GroupPreviewGallery'
+import GroupsGallery from './GroupsGalleryUI'
```

Comment:
I would call it `GroupGallery` as the plural is sort of built into the `Gallery` bit (wouldn't be a gallery with just one... like `GroupList` instead of `GroupsList`), just slightly jars for me as it is now, doesn't flow when I say it very nicely. Not a strong feeling though.

Reply:
Agree, GroupGallery could be enough to figure out the plurality… I just wanted to make extra sure that I don't confuse them with the single-group preview anymore. Will think about it more and then change it later.